src/figuresClasses.ts

```diff
@@ -1,19 +1,81 @@
-export interface Figure {
-
+enum Shape {
+  Triangle = 'triangle',
+  Circle = 'circle',
+  Rectangle = 'rectangle',
 }
 
-export class Triangle {
+type Color = 'red' | 'green' | 'blue';
```

Comment:
you don't need this types, what if color will be pink?

Reply:
Thanks for review.
But in task is written about 3 specific colors. I will wait for mentor opinion))

Reply:
yes, it's true. Colors are limited to these 3. You can use type as you did, or use enum instead